packages/eslint-plugin-pf-codemods/src/rules/v6/kebabToggleReplaceWithMenuToggle/kebabToggle-replace-with-menuToggle.test.ts

```diff
@@ -0,0 +1,188 @@
+    {
+      code: `import { KebabToggle } from "@patternfly/react-core/deprecated"; <KebabToggle></KebabToggle>`,
+      output: `import { MenuToggle } from "@patternfly/react-core";\nimport EllipsisVIcon from "@patternfly/react-icons/dist/esm/icons/ellipsis-v-icon"; <MenuToggle variant="plain" icon={<EllipsisVIcon aria-hidden="true" />}></MenuToggle>`,
+      errors: [
+        {
+          message,
+          type: "ImportDeclaration",
+        },
+        {
+          message,
+          type: "JSXElement",
+        },
+      ],
+    },
+    {
+      code: `import { KebabToggle } from "@patternfly/react-core/deprecated"; <KebabToggle onToggle={someCallback} />`,
+      output: `import { MenuToggle } from "@patternfly/react-core";\nimport EllipsisVIcon from "@patternfly/react-icons/dist/esm/icons/ellipsis-v-icon"; <MenuToggle variant="plain" icon={<EllipsisVIcon aria-hidden="true" />} onClick={someCallback} />`,
+      errors: [
+        {
+          message,
+          type: "ImportDeclaration",
+        },
+        {
+          message,
+          type: "JSXElement",
+        },
+      ],
+    },
+    // The EllipsisVIcon import will be removed by no-unused-imports rule
+    // Possible TODO of only importing this icon if the KebabToggle does not have an icon prop already
+    {
+      code: `import { KebabToggle } from "@patternfly/react-core/deprecated"; <KebabToggle icon={<SomeIcon />} />`,
+      output: `import { MenuToggle } from "@patternfly/react-core";\nimport EllipsisVIcon from "@patternfly/react-icons/dist/esm/icons/ellipsis-v-icon"; <MenuToggle variant="plain" icon={<SomeIcon />} />`,
+      errors: [
+        {
+          message,
+          type: "ImportDeclaration",
+        },
+        {
+          message,
+          type: "JSXElement",
+        },
+      ],
+    },
+    {
+      code: `import { KebabToggle } from "@patternfly/react-core/deprecated"; const Component = KebabToggle;`,
+      output: `import { MenuToggle } from "@patternfly/react-core";\nimport EllipsisVIcon from "@patternfly/react-icons/dist/esm/icons/ellipsis-v-icon"; const Component = MenuToggle;`,
+      errors: [
+        {
+          message,
+          type: "ImportDeclaration",
+        },
+        {
+          message,
+          type: "Identifier",
+        },
+      ],
+    },
+    {
+      code: `import { KebabToggle } from "@patternfly/react-core/dist/esm/deprecated/components/Dropdown/index.js";`,
+      output: `import { MenuToggle } from "@patternfly/react-core";\nimport EllipsisVIcon from "@patternfly/react-icons/dist/esm/icons/ellipsis-v-icon";`,
+      errors: [
+        {
+          message,
+          type: "ImportDeclaration",
+        },
+      ],
+    },
+    {
+      code: `import { KebabToggle } from "@patternfly/react-core/dist/js/deprecated/components/Dropdown/index.js";`,
+      output: `import { MenuToggle } from "@patternfly/react-core";\nimport EllipsisVIcon from "@patternfly/react-icons/dist/esm/icons/ellipsis-v-icon";`,
```

Comment:
🤔 this may or may not really be an issue, but I think we should import the icon from /js if the consumer is importing stuff from /js already.

We may also need to reevaluate if importing from /esm by default is a safe assumption.... but I wouldn't block over that.

Reply:
Made #705 as a followup